fenix/app/src/main/java/org/mozilla/fenix/compose/ext/Modifier.kt

```diff
@@ -49,3 +56,30 @@ fun Modifier.dashedBorder(
         )
     },
 )
+
+/**
+ * Used when clickable needs to be debounced to prevent rapid successive clicks
+ * from calling the onClick function.
+ *
+ * @param debounceInterval The length of time to wait between click events
+ * @param onClick Callback for when item this modifier effects is clicked
+ */
+fun Modifier.debouncedClickable(
+    debounceInterval: Long,
```

Comment:
nit: I think we should specify units for this parameter and we can also have a sane default like 300 or 500 millis.

Reply:
sounds good, I'll add a default.
unfortunately setting the default minimum to anything less than 900 milliseconds seems to not prevent the immediate second click from being registered and sent a `onClick` event.